File: cf/go/26b/main.go
package main

import (
	"fmt"
	"bufio"
	"io"
	// "os"
)

// Input1
// (()))(
// Ouput1
// 4

func solve(_r io.Reader, _w io.Writer) {
	in := bufio.NewReader(_r)
	out := bufio.NewWriter(_w)
	defer out.Flush()

	var s string
	fmt.Fscan(in, &s)
	st := 0
	ans := 0
	for i := range s {
		if s[i] == '(' {
			st++
		} else {
			if st > 0 {
				st--
				ans++
			}
		}
	}
	fmt.Fprintln(out, ans * 2)
}

File: cf/py/26b/main.py
def solve():
    s = input()
    ans = 0
    st = 0
    for c in s:
        if c == '(':
            st += 1
        else:
            if st:
                st -= 1
                ans += 1
    print(ans * 2)
if __name__ == "__main__":
    solve()   